.circleci/config.yml

```diff
@@ -23,6 +23,18 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: deploy
```

Comment:
could we change the name here to smth like "trigger deploy", had really hard time figuring out what is the difference between this and the bottom change :)